test/test_transforms.jl

```diff
@@ -5,7 +5,7 @@ using Base.Test
 
 using Celeste: Types, SampleData, Transform, SensitiveFloats
 import Celeste: ModelInit, ElboDeriv
-
+using Compat
```

Comment:
Sorry for  removing all the `using Compat`s in #134! I figured most were for 0.3->0.4 changes and thus no longer needed.